### PR TITLE
Enable termination message log in Hyper runtime

### PR DESCRIPTION
**Pull request by resouer**
1. mount termination msg path as Hyper volume
2. add msg path info into Pod label (we don't have container level label/annotation yet)
3. when pod run into `StatusSuccess`, collect the msg from msg path

Fixes: #93

Usage:

```
// After the container exited
kubectl get pods/pod-w-message  -o yaml

...
 state:
      terminated:
        containerID: hyper://852a3c522c4c5e4caf0c14b20dc9ee1c24ef6a2d1d78cd1e33fe4256c2373a0d
        exitCode: 0
        finishedAt: 2016-04-22T12:24:09Z
        message: |
          Sleep expired
        reason: Succeeded
        startedAt: 2016-04-22T12:23:38Z
...
```

### CODE REVIEW

File: pkg/kubelet/hyper/hyper.go [hunk 1]
```diff
@@ -245,10 +245,20 @@ func (r *runtime) getContainerStatus(container ContainerStatus, image, imageID,
 			status.FinishedAt = terminatedFinishedAt
 		}
 
+		var message string
+		// (TODO) harryz only work when added time.Sleep()
+		time.Sleep(1 * time.Second)
```
Comment (resouer) [inline pkg/kubelet/hyper/hyper.go]: @feiskyer Any idea about why it has to sleep before read file?

File: pkg/kubelet/hyper/hyper.go [hunk 2]
```diff
@@ -526,6 +534,39 @@ func (r *runtime) buildHyperPod(pod *api.Pod, restartCount int, pullSecrets []ap
...
+				Name:          fmt.Sprintf("termination-message-%s", randomUID),
+				ContainerPath: container.TerminationMessagePath,
+				HostPath:      containerLogPath,
+				ReadOnly:      false,
+			}
+			opts.Mounts = append(opts.Mounts, *mnt)
+
+			// set termination msg labels with host path
+			terminationMsgLabels[container.Name] = mnt.HostPath
+			glog.Errorf("Harry termination host path %v", terminationMsgLabels)
```
Comment (feiskyer) [inline pkg/kubelet/hyper/hyper.go]: This is only for debug?

Comment (resouer) [inline pkg/kubelet/hyper/hyper.go]: Sorry, forget to delete it.

**Comment by feiskyer** [comment]
lgtm

**Pull request merged**